magma-feedstock/recipe/meta.yaml

```diff
@@ -24,9 +26,13 @@ requirements:
     - patch       # [not win]
   host:
     - cudatoolkit {{ cudatoolkit }}*
-    - mkl
+    - intel-openmp {{ mkl }}
+    - mkl {{ mkl }}
+    - mkl-include
   run:
     - {{ pin_compatible('cudatoolkit', max_pin='x.x') }}
+    - intel-openmp {{ mkl }}
```

Comment:
I'm curious, if you do a `conda search -i magma -c local --override-channels`, what do the dependencies look like? I've seen conda-build do weird things when using `{{ <var> }}` in the runtime dependencies.

Reply:
What sort of weirdness have you encountered? This is a fairly common thing to do and since it's resolved at build-time it's really no different than specifying the version yourself. In this case, the dependencies would look something like:
```
- mkl 2021.*
- intel-openmp 2021.*
- cudatoolkit >=11.8.0,<11.9.0a0
```

Reply:
yes, they're
```
  - cudatoolkit >=11.8.0,<11.9.0a0
  - intel-openmp 2023.*
  - libgcc-ng >=11.2.0
  - libstdcxx-ng >=11.2.0
  - mkl 2023.*
 ```